lessons/day6/arrays_vs_hashes.md

```diff
@@ -1,79 +1,88 @@
 _JumpStart Live (JSL)_
+
 # Day 6
+
 ## Arrays vs Hashes
 
 ### Arrays
+
 Arrays are ordered lists of items. The order of the data is maintained and important. Arrays have integer indexes starting at 0 which cannot be changed. This means you can retrieve data using a 0-based index.
 
 ### Hashes
+
 Hashes are a collection of paired information. Also described as a collection of key-value pairs, or as a "look up table". Hashes use any object for their index.
 
 Each pair has both a key and a value. The keys in the hash are unique (i.e., not repeated), but values may be repeated. Hashes store information in the order in which the keys were inserted, you however should never depend on this order.
 
 ### Similarities
+
 * Both Arrays and Hashes are collections that store and retrieve data
 * If you try and access an element of an array or a hash that does not exist, the value `nil` will be sent back
 
 ### Differences
+
 * Arrays store values, but have fixed indexes (integers, starting at 0)
 * Hashes allow you to still store data but to define the index as any object
 
 ### Tips
+
 If you want to store single pieces of information use an array. If your data is better described as a pair, then use a Hash.
 
 ### Examples
+
 1. You want to store the names of all the students in Ada, class of C7.
 
-	```ruby
-	names = ["Riley", "Sam", "Alex"]
-	```
+```ruby
+names = ["Riley", "Sam", "Alex"]
+```
```

Comment:
Huh, interesting. I thought the extra spaces was just a syntactical error, but I tested it locally and turns out having four spaces of indentation at the beginning aligns the code block to under the bullet point.

---

1. Without indenting
```ruby
# it's not aligned
```

1. with indenting
    ```ruby
    # it *is* aligned
    ````